iOS/second-hand/second-hand/Home/subScene/RegisterNewProduct/RegisterNewProductViewController.swift

```diff
@@ -0,0 +1,200 @@
+        let appearance = UIToolbarAppearance()
+            appearance.configureWithOpaqueBackground()
+            appearance.backgroundColor = .neutralBackgroundWeak
+            navigationController?.toolbar.scrollEdgeAppearance = appearance
+    }
+    
+    private func layout() {
+        let sectionArr = [sectionLine1, sectionLine2, sectionLine3, square, imageLabelStackView, titleTextField, priceTextField, descriptionTextField, wonIcon]
+        sectionArr.forEach{
+            self.view.addSubview($0)
+            $0.translatesAutoresizingMaskIntoConstraints = false
+        }
+        NSLayoutConstraint.activate([
+            
+            sectionLine1.topAnchor.constraint(equalTo: self.view.topAnchor, constant: 167),
+            sectionLine1.centerXAnchor.constraint(equalTo: self.view.centerXAnchor),
+            
+            
+            sectionLine2.topAnchor.constraint(equalTo: sectionLine1.bottomAnchor, constant: 52),
+            sectionLine2.centerXAnchor.constraint(equalTo: self.view.centerXAnchor),
+            
+            sectionLine3.topAnchor.constraint(equalTo: sectionLine2.bottomAnchor, constant: 52),
+            sectionLine3.centerXAnchor.constraint(equalTo: self.view.centerXAnchor),
+            
+            square.bottomAnchor.constraint(equalTo: sectionLine1.topAnchor, constant: -15),
+            square.leadingAnchor.constraint(equalTo: self.view.leadingAnchor, constant: 15),
+            
+            imageLabelStackView.centerXAnchor.constraint(equalTo: square.centerXAnchor),
+            imageLabelStackView.centerYAnchor.constraint(equalTo: square.centerYAnchor),
+            
+            titleTextField.leadingAnchor.constraint(equalTo: square.leadingAnchor),
+            titleTextField.topAnchor.constraint(equalTo: sectionLine1.bottomAnchor, constant: 15),
+            titleTextField.bottomAnchor.constraint(equalTo: sectionLine2.topAnchor, constant: -15),
+            
+            wonIcon.leadingAnchor.constraint(equalTo: square.leadingAnchor),
+            wonIcon.topAnchor.constraint(equalTo: sectionLine2.bottomAnchor, constant: 15),
+            wonIcon.bottomAnchor.constraint(equalTo: sectionLine3.topAnchor, constant: -15),
+            
+            priceTextField.leadingAnchor.constraint(equalTo: wonIcon.trailingAnchor, constant: 3),
+            priceTextField.topAnchor.constraint(equalTo: sectionLine2.bottomAnchor, constant: 15),
+            priceTextField.bottomAnchor.constraint(equalTo: sectionLine3.topAnchor, constant: -15),
+            
+            descriptionTextField.leadingAnchor.constraint(equalTo: square.leadingAnchor),
+            descriptionTextField.trailingAnchor.constraint(equalTo: self.view.trailingAnchor, constant: -15),
+            descriptionTextField.topAnchor.constraint(equalTo: sectionLine3.bottomAnchor, constant: 15),
+            descriptionTextField.bottomAnchor.constraint(equalTo: self.view.bottomAnchor, constant: -15)
```

Comment:
텍스트뷰의 바텀이 툴바와 관계없이 view의 바텀으로 잡혀있습니다.
레이아웃을 잡을때는 [SafeArea](https://developer.apple.com/documentation/uikit/uiview/positioning_content_relative_to_the_safe_area)를 고려해야 합니다.

```suggestion
            descriptionTextField.bottomAnchor.constraint(equalTo: self.view.safeAreaLayoutGuide.bottomAnchor, constant: -15)
```

Reply:
정답이 있다니 이건 귀하군요.

Reply:
수정했더니 해결되었습니다 감사합니다 !! 👍👍